randovania/games/prime1/logic_database/Phazon Mines.txt

```diff
@@ -402,24 +402,91 @@ Extra - aabb: [-36.12424, -29.465834, -4.4074636, 29.15716, 66.50636, 24.451797]
       All of the following:
           After Mine Security Station Unlock Doors
           Any of the following:
-              Combat (Intermediate) or Normal Damage ≥ 125
-              Shoot Ice Beam and Shoot Wave Beam
+              # Avoid Pirates
+              Combat (Expert) or Normal Damage ≥ 199
+              All of the following:
+                  # Fight Pirates
+                  Shoot Ice Beam and Shoot Wave Beam
+                  Any of the following:
+                      Combat (Advanced) or Normal Damage ≥ 149
+                      All of the following:
+                          Charge Beam
+                          Any of the following:
+                              Combat (Intermediate) or Normal Damage ≥ 99
+                              Combat (Beginner) and Normal Damage ≥ 49
   > Door to Security Access B
       All of the following:
           After Mine Security Station Unlock Doors
           Any of the following:
-              Combat (Intermediate) or Normal Damage ≥ 125
-              Shoot Ice Beam and Shoot Wave Beam
+              # Avoid Pirates
+              Combat (Expert) or Normal Damage ≥ 199
+              All of the following:
+                  # Fight Pirates
+                  Charge Beam and Combat (Advanced) and Normal Damage ≥ 149 and Shoot Ice Beam and Shoot Wave Beam
+                  Any of the following:
+                      Combat (Intermediate) or Normal Damage ≥ 99
+                      Combat (Beginner) and Normal Damage ≥ 49
   > Door to Storage Depot A
       After Mine Security Station Barrier
   > Event - Mine Security Station Barrier Opened
-      Morph Ball and Power Bomb and Scan Visor
+      Any of the following:
+          All of the following:
+              # Avoid Wave Pirates
+              Before Mine Security Station Unlock Doors
+              Any of the following:
+                  Any of the following:
+                      # Avoid All Pirates
+                      Combat (Hypermode) or Normal Damage ≥ 299
+                      All of the following:
+                          # Fight Shadow Pirates
+                          Shoot Any Beam
+                          Any of the following:
+                              Combat (Advanced) or Normal Damage ≥ 199
+                              All of the following:
+                                  Combat (Intermediate)
+                                  Any of the following:
+                                      Normal Damage ≥ 99
+                                      Charge Beam and Normal Damage ≥ 49
```

Comment:
It's even worse actually, worst case scenario you're requiring the player to have *4 fights* worth of energy just to make it through the first pass and collect the item.

Reply:
I don't quite understand what you mean here. 

- `Security Access A -> Room Center` is trivial 
-`Room Center -> Event - Mine Security Station Barrier Opened` has combat requirements
- `Event - Mine Security Station Barrier Opened -> Room Center` is trivial
-`Room Center -> Storage Depot A` is practically trivial
-`Storage Depot A -> Room Center` is trivial, and connections to the doors from `Room Center` have combat requirements.

At most I'm seeing 318 energy for strict requirements. Could you explain what I'm missing?

Reply:
My fault. I rushed the review and misread the logic.

Thanks for your patience